noetic/distribution.yaml

```diff
@@ -5592,6 +5592,24 @@ repositories:
       type: git
       url: https://github.com/221eROS/mitch_v2_driver.git
       version: main
+  mlx90640_thermal_camera:
+    doc:
+      type: git
+      url: https://github.com/vakshit/mlx90640_thermal_camera.git
+      version: master
+    release:
+      packages:
+      - mlx90640-thermal-camera
```

Comment:
This is fairly unusual.  The name of the package as specified in https://github.com/vakshit/mlx90640_thermal_camera/blob/master/package.xml is `mlx90640_thermal_camera`, but here it has the dashes instead of the underscores.  Do you know how that happened?

Reply:
Actually, i followed the documentation on ROS Wiki. The package is meant to be installed as ros-noetic-mlx90640-thermal-camera.

Reply:
> Actually, i followed the documentation on ROS Wiki. The package is meant to be installed as ros-noetic-mlx90640-thermal-camera.

That makes sense, but actually the buildfarm will take care of that bit.  The package name here should match what is in the package.xml (in this case, use underscore instead of dashes).

Reply:
so, a commit changing that name suffices.?

Reply:
> so, a commit changing that name suffices.?

I think that should work.

Reply:
OK, it turns out I was wrong.  You'll need to re-bloom in order to make this work, because otherwise the wrong branches and package names are in the release repository.